### ISSUE TITLE
Issue with Auto_Curseforge after mod update "not one of the values accepted for Enum class"

### ISSUE THREAD

**Issue by corydlamb** [itzg/docker-minecraft-server#2315]
### Describe the problem

When I try and run my Auto_Cursforge Docker container (as of today, there was an update for this mod and it seems to have broken something) I get the following output from the container. Looks like it has something to do with the modloadertype being passed from the API (as far as my untechnical brain can tell lol).

### Container definition

```
Used portainer on the stable tag with the following env variables

CF_API_KEY | secret
-- | --
CF_PAGE_URL | https://www.curseforge.com/minecraft/modpacks/darkrpg
EULA | true
GID | 1000
JAVA_HOME | /opt/java/openjdk
JAVA_VERSION | jdk-19.0.2+7
LANG | en_US.UTF-8
LANGUAGE | en_US:en
LC_ALL | en_US.UTF-8
MEMORY | 12G
PATH | /opt/java/openjdk/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
TYPE | AUTO_CURSEFORGE
TZ | America/New_York
UID | 1000
VERSION | 1.20.1
```

### Container logs

```
[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 21 1000 1000 4096 Aug  2 21:39 /data'
[mc-image-helper] 21:45:51.666 WARN  : Some files from modpack file DarkRPG-7.1.6.zip were missing. Proceeding with a re-install
[mc-image-helper] 21:45:52.327 INFO  : Processing modpack 'DarkRPG-7.1.6.zip' (darkrpg) @ 515345:4678678
[mc-image-helper] 21:45:52.739 ERROR : 'install-curseforge' command failed. Version is 1.32.8
me.itzg.helpers.errors.GenericException: Failed to parse response body into class me.itzg.helpers.curseforge.model.GetModResponse: Cannot deserialize value of type `me.itzg.helpers.curseforge.model.ModLoaderType` from String "6": not one of the values accepted for Enum class: [0, 1, 2, 3, 4, 5]
 at [Source: (reactor.netty.ByteBufMono$ReleasingInputStream); line: 1, column: 13439] (through reference chain: me.itzg.helpers.curseforge.model.GetModResponse["data"]->me.itzg.helpers.curseforge.model.CurseForgeMod["latestFilesIndexes"]->java.util.ArrayList[1]->me.itzg.helpers.curseforge.model.FileIndex["modLoader"])
	at me.itzg.helpers.http.ObjectFetchBuilder.lambda$handleResponse$1(ObjectFetchBuilder.java:89)
	at reactor.core.publisher.MonoFlatMap$FlatMapMain.onNext(MonoFlatMap.java:132)
	at reactor.core.publisher.MonoPublishOn$PublishOnSubscriber.run(MonoPublishOn.java:181)
	at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:68)
	at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:28)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
	Suppressed: java.lang.Exception: #block terminated with an error
		at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:102)
		at reactor.core.publisher.Mono.block(Mono.java:1712)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.processModpack(CurseForgeInstaller.java:507)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.resolveModpackFileAndProcess(CurseForgeInstaller.java:328)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.installByRetrievingModpackZip(CurseForgeInstaller.java:196)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.lambda$install$4(CurseForgeInstaller.java:130)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.install(CurseForgeInstaller.java:165)
		at me.itzg.helpers.curseforge.CurseForgeInstaller.install(CurseForgeInstaller.java:129)
		at me.itzg.helpers.curseforge.InstallCurseForgeCommand.call(InstallCurseForgeCommand.java:202)
		at me.itzg.helpers.curseforge.InstallCurseForgeCommand.call(InstallCurseForgeCommand.java:28)
		at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
		at picocli.CommandLine.access$1500(CommandLine.java:148)
		at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
		at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
		at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
		at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
		at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
		at picocli.CommandLine.execute(CommandLine.java:2170)
		at me.itzg.helpers.McImageHelper.main(McImageHelper.java:149)
Caused by: com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `me.itzg.helpers.curseforge.model.ModLoaderType` from String "6": not one of the values accepted for Enum class: [0, 1, 2, 3, 4, 5]
 at [Source: (reactor.netty.ByteBufMono$ReleasingInputStream); line: 1, column: 13439] (through reference chain: me.itzg.helpers.curseforge.model.GetModResponse["data"]->me.itzg.helpers.curseforge.model.CurseForgeMod["latestFilesIndexes"]->java.util.ArrayList[1]->me.itzg.helpers.curseforge.model.FileIndex["modLoader"])
	at com.fasterxml.jackson.databind.exc.InvalidFormatException.from(InvalidFormatException.java:67)
	at com.fasterxml.jackson.databind.DeserializationContext.weirdStringException(DeserializationContext.java:2002)
	at com.fasterxml.jackson.databind.DeserializationContext.handleWeirdStringValue(DeserializationContext.java:1230)
	at com.fasterxml.jackson.databind.deser.std.EnumDeserializer._deserializeAltString(EnumDeserializer.java:415)
	at com.fasterxml.jackson.databind.deser.std.EnumDeserializer._fromString(EnumDeserializer.java:279)
	at com.fasterxml.jackson.databind.deser.std.EnumDeserializer.deserialize(EnumDeserializer.java:258)
	at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:129)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:314)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer._deserializeFromArray(CollectionDeserializer.java:359)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:244)
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:28)
	at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:129)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:314)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
	at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:129)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:314)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177)
	at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:323)
	at com.fasterxml.jackson.databind.ObjectReader._bindAndClose(ObjectReader.java:2105)
	at com.fasterxml.jackson.databind.ObjectReader.readValue(ObjectReader.java:1481)
	at me.itzg.helpers.http.ObjectFetchBuilder.lambda$handleResponse$1(ObjectFetchBuilder.java:85)
	... 9 common frames omitted
[init] ERROR failed to auto-install CurseForge modpack
```

**Comment by itzg** [comment]
This was already fixed in #2308. Pull the latest image and try again.

**Comment by corydlamb** [comment]
> This was already fixed in #2308. Pull the latest image and try again.

Many thanks!
